operator/v1/types_network.go

```diff
@@ -473,25 +473,61 @@ type NetFlowConfig struct {
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=10
 	Collectors []IPPort `json:"collectors,omitempty"`
+
+	// flowCollectors defines the NetFlow collectors that will consume the flow data exported from OVS.
+	// It is a list of objects featuring a valid IP address and a port with a maximum of ten items
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:MaxItems=10
+	FlowCollectors []NetworkFlowTarget `json:"flowCollectors,omitempty"`
 }
 
 type SFlowConfig struct {
 	// sFlowCollectors is list of strings formatted as ip:port with a maximum of ten items
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=10
 	Collectors []IPPort `json:"collectors,omitempty"`
+
+	// flowCollectors defines the NetFlow collectors that will consume the flow data exported from OVS.
+	// It is a list of objects featuring a valid IP address and a port with a maximum of ten items
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:MaxItems=10
+	FlowCollectors []NetworkFlowTarget `json:"flowCollectors,omitempty"`
 }
 
 type IPFIXConfig struct {
 	// ipfixCollectors is list of strings formatted as ip:port with a maximum of ten items
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=10
 	Collectors []IPPort `json:"collectors,omitempty"`
+
+	// flowCollectors defines the NetFlow collectors that will consume the flow data exported from OVS.
+	// It is a list of objects featuring a valid IP address and a port with a maximum of ten items
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:MaxItems=10
+	FlowCollectors []NetworkFlowTarget `json:"flowCollectors,omitempty"`
 }
 
 // +kubebuilder:validation:Pattern=`^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]):([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$`
 type IPPort string
```

Comment:
do we block system UDP ports anywhere?  Or ports < 1024?

```
UNCONN 0      0                 0.0.0.0:111        0.0.0.0:*    users:(("rpcbind",pid=2076,fd=5),("systemd",pid=1,fd=89))
UNCONN 0      0               127.0.0.1:323        0.0.0.0:*    users:(("chronyd",pid=1117,fd=6))
UNCONN 0      0               127.0.0.1:1001       0.0.0.0:*    users:(("rpc.statd",pid=2095,fd=9))
UNCONN 0      0                 0.0.0.0:6081       0.0.0.0:*
UNCONN 0      0                 0.0.0.0:57347      0.0.0.0:*    users:(("rpc.statd",pid=2095,fd=8))
```

Reply:
We're not doing that for the previous IPv4 exported flows, are we ? 

Not saying we shouldn't - but we should have a consistent behavior across both IP families IMO.

As such, if we choose to implement that, I think we should address it for both families, but in a follow-up PR.